src/Dropdown/__tests__/Dropdown-test.js

```diff
@@ -19,6 +19,41 @@ if (React.version.match(/15.[0-5]/)) {
 var MockDropdownList = require("./fixtures/MockDropdownList");
 var Dropdown = require("../Dropdown.js");
 
+class InputTrigger extends React.Component {
```

Comment:
Why do we declare a new class in this test instead of using an actual component from `./DropdownTriggers`?